### PR TITLE
bug 1258038: add DNT lists to stage.ini config

### PR DESCRIPTION
**Pull request by groovecoder**

### CODE REVIEW

**Comment by groovecoder** [comment]
I think this is how we'll deploy the new list config to stage.

**Comment by ckolos** [comment]
are these "stage-only" forever, or "stage for now, one day, prod"

**Comment by groovecoder** [comment]
Stage for now, one day: prod.

File: stage.ini [hunk 1]
```diff
@@ -13,6 +13,36 @@ disconnect_url=https://raw.githubusercontent.com/mozilla-services/shavar-test-li
...
+[tracking-protection-baseeff]
+disconnect_url=https://raw.githubusercontent.com/mozilla-services/shavar-prod-lists/master/disconnect-blacklist.json
+output=baseeff-track-digest256
+
+# DNT="W3C", all categories except content category
+[tracking-protection-basew3c]
+disconnect_url=https://raw.githubusercontent.com/mozilla-services/shavar-prod-lists/master/disconnect-blacklist.json
+output=basew3c-track-digest256
+
+# DNT="", content category
```
Comment (fmarier) [inline stage.ini]: As I mentioned on https://github.com/mozilla-services/shavar-list-creation/pull/37/files#r66558758, we could add "content category only" to these to make it clearer.

**Comment by fmarier** [comment]
r+ with or without the comment tweak I suggested.

**Comment by groovecoder** [comment]
@fmarier - I added `s3_key` values for the DNT lists. I followed the pattern of the existing lists. r?

**Comment by fmarier** [comment]
The S3 names look good to me. r+.

**Comment by ckolos** [comment]
I'd like to wait on merging this to the master branch until QA has had a chance to do end to end testing of the new lists in stage with the up-coming deploy. We're currently using this branch to generate the lists in stage and there's been no issues, but if this is merged, those lists and code changes will appear in prod without the benefit of a full QA cycle. 

If I'm mistaken on that, please feel free to explain. Pending that QA cycle, however, r+.

**Comment by groovecoder** [comment]
Thanks :ckolos

**Comment by groovecoder** [comment]
Updated branch with https://github.com/mozilla-services/shavar-list-creation-config/pull/11/commits/f6b8c3711f6ef62ba50caabd0b3c39e2040cf124 to remove mozpub-track from stage.ini

**Pull request merged**